packages/jira-adapter/test/filters/automation/automation_label/label_fetch.test.ts

```
@@ -207,63 +179,6 @@ describe('automationLabelFetchFilter', () => {
       expect(automation.elemID.getFullName()).toEqual('jira.AutomationLabel.instance.someName')
     })
 
-    it('should throw if resources response is invalid', async () => {
-      connection.post.mockImplementation(async url => {
-        if (url === '/rest/webResources/1.0/resources') {
-          return {
-            status: 200,
-            data: {
-              unparsedData: {},
-            },
-          }
-        }
-
-        throw new Error(`Unexpected url ${url}`)
-      })
```

Keep one test to make sure the failure is not swallowed for labels

---

There is already a test for client that checks whether an invalid response to getCloudId throws.

---

But what if label_fetch ignores it or does not handle it properly?

---

I think you missed this comment

---

Re-added one test.